assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -146,30 +212,77 @@ class TextBlockEdit extends Component {
 				</BlockControls>
 				<div className={ classnames( wrapperClass, {
 					'with-line-height': ampFitText,
+					'is-empty-draggable-text': ! isEditing && ! content.length,
 				} ) } style={ wrapperStyle } >
-					<RichText
-						wrapperClassName="wp-block-amp-story-text"
-						tagName="p"
-						// Ensure line breaks are normalised to HTML.
-						value={ content }
-						onChange={ ( nextContent ) => setAttributes( { content: nextContent } ) }
-						// The 2 following lines are necessary for pasting to work.
-						onReplace={ this.onReplace }
-						onSplit={ () => {} }
-						style={ {
-							color: textColor.color,
-							fontSize: ampFitText ? autoFontSize + 'px' : userFontSize,
-							textAlign: align,
-							position: ampFitText && content.length ? 'static' : undefined,
-						} }
-						className={ classnames( className, {
-							'has-text-color': textColor.color,
-							[ textColor.class ]: textColor.class,
-							[ fontSize.class ]: ampFitText ? undefined : fontSize.class,
-							'is-amp-fit-text': ampFitText,
-						} ) }
-						placeholder={ placeholder || __( 'Write text…', 'amp' ) }
-					/>
+					{ isEditing &&
+						<RichText
+							wrapperClassName={ textWrapperClassName }
+							tagName="p"
+							// Ensure line breaks are normalised to HTML.
+							value={ content }
+							onChange={ ( nextContent ) => setAttributes( { content: nextContent } ) }
+							// The 2 following lines are necessary for pasting to work.
+							onReplace={ this.onReplace }
+							onSplit={ () => {} }
+							style={ textStyle }
+							className={ classnames( className, textClassNames ) }
+							placeholder={ placeholder || __( 'Write text…', 'amp' ) }
+						/>
+					}
+					{ ! isEditing &&
+						<StoryBlockMover
+							clientId={ clientId }
+							blockName={ name }
+							blockElementId={ `block-${ clientId }` }
+							isDraggable={ ! isPartOfMultiSelection }
+							isMovable={ true }
+						>
+							<div
+								role="textbox"
+								tabIndex="-1"
+								className="is-not-editing editor-rich-text block-editor-rich-text wp-block-amp-story-text"
+								onClick={ () => {
+									if ( isSelected ) {
+										this.toggleIsEditing( true );
+									}
+								} }
+								onKeyDown={ ( e ) => {
+									if ( ENTER === e.keyCode && isSelected ) {
+										this.toggleIsEditing( true );
+									}
+								} }
+								onMouseDown={ ( event ) => {
+									// Prevent text selection on double click.
+									if ( 1 < event.detail ) {
+										event.preventDefault();
+									}
+								} }
+							>
+								{ hasOverlay && (
+									<div
+										role="textbox"
+										tabIndex="-1"
+										className="amp-overlay"
+										onClick={ ( e ) => {
+											this.toggleOverlay( false );
+											e.stopPropagation();
+										} }
+										onKeyDown={ () => {} }
```

Comment:
That's here for linting purposes only, is there anything useful we could actually add here? `ENTER` could be useful, however, it doesn't really seem to be working.

Reply:
Which linter rule is causing this then? An a11y one?

Working `ENTER` support would be nice.

Also: `ESC` to remove editing mode again

Reply:
^ @miina thoughts on supporting this? Would be good to have keyboard support here.

Reply:
Same issue here as above -- the events are not triggered.

Reply:
Added `ENTER` support, `ESC` support wouldn't make sense here since this element is not displayed in editing mode at all. `ESCAPE` support for `RichText` doesn't work either since it already has some kind of handling for that.